Makefile

```diff
@@ -52,7 +52,7 @@ install-linter: ## Installs the linter
 
 .PHONY: lint
 lint: ## Runs the linter
-	$$(go env GOPATH)/bin/golangci-lint run
+	export "GOROOT=$$(go env GOROOT)" && $$(go env GOPATH)/bin/golangci-lint run
```

Comment:
my linter is still failing. Do you know why?

Reply:
If so, we must change the `make install-linter` to match this new version.
And yes, I can confirm the 1.46.2 version was working on my Windows/WSL/Ubuntu machine, but it doesn't work on my mac, updating it to 1.52.2 fixed the problem.

Reply:
I've updated the `make install-linter`